src/Type/Enum/UsersConnectionSearchColumnEnum.php

```diff
@@ -6,30 +6,34 @@
 
 class UsersConnectionSearchColumnEnum {
 	public static function register_type() {
-		global $wp_roles;
```

Comment:
I wonder if we should add this in another enum type. Probably a simpler path for migration from folks using the incorrect way.